conf/ftp

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+# Copying cert.pem and dhparams.pem to pure-ftpd.pem and pure-ftpd-dhparams.pem.
+cp /etc/ssl/private/cert.pem /etc/ssl/private/pure-ftpd.pem
+cp /etc/ssl/private/dhparams.pem /etc/ssl/private/pure-ftpd-dhparams.pem
+
+# Remove root from users that cannot login into the system via FTP
+sed -i "s|^root$|# root|g"  /etc/ftpusers
+
+# Configure settings for pure-ftpd
+echo yes > /etc/pure-ftpd/conf/ChrootEveryone
+echo yes > /etc/pure-ftpd/conf/Daemonize
+echo no > /etc/pure-ftpd/conf/NoAnonymous
+echo 0 > /etc/pure-ftpd/conf/MinUID
+echo yes > /etc/pure-ftpd/conf/AllowUserFXP
+echo 1 > /etc/pure-ftpd/conf/TLS
+echo HIGH:MEDIUM:+TLSv1:\!SSLv2:\!SSLv3 > /etc/pure-ftpd/conf/TLSCipherSuite
```

Comment:
Whilst this looks pretty secure, we have a [common conf script](https://github.com/turnkeylinux/common/blob/master/conf/turnkey.d/zz-ssl-ciphers) so we only need to update ciphers in one single place and then everything else gets them. I'd prefer it if the pure-ftpd config leveraged this same thing - that will reduce the risk that it might get overlooked in a future update.

Reply:
Have added the following block to `zz-ssl-ciphers` file

```
# Pure-FTPd
CONF="/etc/pure-ftpd/pure-ftpd.conf"
if [ -f "$CONF" ]; then
    sed -i "/^#\{0,1\} \{0,\}TLS    / s|.*TLS\(  *\)[0-9][0-9]*|TLS\11|" $CONF
    sed -i "/^#\{0,1\} \{0,\}TLSCipherSuite    / s|.*TLSCipherSuite\(  *\)..*|TLSCipherSuite\1HIGH:MEDIUM:+TLSv1:\!SSLv2:\!SSLv3|" $CONF
    
    sed -i "/^#\{0,1\} \{0,\}CertFile    / s|.*CertFile\(  *\)..*|CertFile\1$CERTFILE|" $CONF
fi
```

This changes values in `.conf` files, in the following way 
- `TLS` is changed to `1` (which accepts secure FTP as well as insecure plain FTP)
- `TLSCipherSuite` is changed to `HIGH:MEDIUM:+TLSv1:!SSLv2:!SSLv3`
- `CertFile` is uncomment'ed to enable it, and `$CERTFILE` path is used (from the variable's value declared above).

Reply:
Ok, that seems pretty good. Although TLS1 & TLS1.1 should also be disabled.

Reply:
I will check what can be done on disabling  TLS1 & TLS1. 1.

Reply:
I have successfully disabled TLS1 and TLS1.1 as well, and created options for both the methods `linking method` as well as `.conf file method` that i have mentioned earlier. I have used the testssl.sh to confirm (PFA attachment for the same).


![testssl sh](https://github.com/turnkeylinux/common/assets/3785927/0f246b54-943c-431e-9472-ab1f1b41cc74)

Reply:
@JedMeister Reply awaited.

Reply:
I don't see any updates? Have you pushed them?

Reply:
@JedMeister Not yet, as i wanted your advise over it as i mentioned here
https://github.com/turnkeylinux/common/pull/247#discussion_r1282006596
> @JedMeister
> 
>     I have tested using the `linking method`, creating symbolic link of cert.pem and dhparams.pem, it is working without any issues.
> 
>     I have also tested using the `.conf file method`, that is also working fine.
> 
> 
> Please do suggest which method do you want me to implement?

Reply:
My apologies. Responded above.